client/src/pages/Upcoming/index.tsx

```diff
@@ -16,6 +16,7 @@ import './style.css';
 import { ApplicationObject, UpcomingObject } from "typings";
 import Loading from "components/Loading";
 import SideDrawer from "components/SideDrawer";
+import Error from 'pages/Error';
```

Comment:
Do you think we should move Error from pages to component since that is how you are using it more?

Reply:
I agree